test/contracts/apps/disputable/disputable_app.js

```diff
@@ -135,14 +135,25 @@ contract('DisputableApp', ([_, owner, agreement, anotherAgreement, someone]) =>
     })
 
     context('when the agreement is set', () => {
+      let agreement
+
       beforeEach('set agreement', async () => {
-        const agreement = await AgreementMock.new()
+        agreement = await AgreementMock.new()
         await disputable.setAgreement(agreement.address, { from: owner })
       })
 
       it('does not revert', async () => {
         await disputable.newAction(0, '0x00', owner)
       })
+
+      it('transfer funds when sent', async () => {
```

Comment:
```suggestion
      it('receives ETH when sent', async () => {
```

Maybe this is a better description? My thought line here is that this test is pretty hard to tell what it's actually testing at the moment, especially out of context.